pulumitest/pulumiTest.go

```diff
@@ -72,7 +71,7 @@ func (a *PulumiTest) Source() string {
 }
 
 // T returns the current testing.T instance.
-func (a *PulumiTest) T() *testing.T {
```

Comment:
This is breaking. Could attempt a downcast here.

Reply:
Yeah - think that's fine - return nil if not possible?

Reply:
Not a fan of returning nils. But let me try to downcast it back and see if it still works for my use in cross-tests in the bridge.